src/data/index.ts

```diff
@@ -321,6 +321,19 @@ export const exportCanvas = async (
       return;
     }
   }
+  if (type === "pdf") {
+    const tempPdf = exportToPdf(elements, appState, {
+      exportBackground,
+      viewBackgroundColor,
+      scale,
+      exportPadding,
+      shouldAddWatermark,
+    });
+    await fileSave(new Blob([tempPdf], { type: "application/pdf" }), {
```

Comment:
for some reason, it doesn't seem to autocomplete it as .pdf on Chrome for me
![image](https://user-images.githubusercontent.com/197597/85312272-796e6600-b46b-11ea-8469-35ce6837bda1.png)